test/StraightLinePrograms/runtests.jl

```diff
@@ -4,9 +4,10 @@ using ..Test
 using ..Oscar
 const SLP = Oscar.StraightLinePrograms
 
-include("setup.jl")
-include("straightline.jl")
-include("gap.jl")
-include("atlas.jl")
+# make sure to use the custom include function to collect stats
+Main.include("setup.jl")
+Main.include("straightline.jl")
+Main.include("gap.jl")
+Main.include("atlas.jl")
```

Comment:
Does this really work? Whenever you create a new `module` then Julia installs a custom `include` method for it of the form `include(p) = Base.include(Mod, p)` where `Mod` is the module (here: `SLPTest`). Source: the docstring for `baremodule`.

So just calling `Main.include` doesn't seem likely to support this...?

Reply:
OK fair enough. I guess as long as this is the last test (and thus cannot affect others tests), and if it really works, then it is fine :-)